### PR TITLE
New doc for Windows users

### PR DESCRIPTION
**Pull request by cwaldron97**
Here is a markdown document for Windows Users currently based on my experience with Windows. I will continue to update this but I wanted to put my thoughts down for now as there are some problems I think I can find solutions to.

### CODE REVIEW

File: docs/windows-users.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+the `/D` option specifies that it is a directory. You will need to run this command in `cmd.exe` instead of `Windows Powershell` as it is not a standalone executable.
+
+## Running a Bitcoind node
+Currently there are written changes in code to make it so you are run a node on Windows. Path specs are based on out of box installation. In the case you receive an error like `Could not locate bitcoind on user PATH`
+then you will need to do some tweaking of either the code or moving your folder into the correct location. Currently this is specified by the `DEFAULT_DATADIR` in `BitcoindConfig` within `Bitcoind-rpc`. When you install bitcoin out of box there are 2 folders
+created, 1 which is contained in the location on the wiki and that contains data generated after syncing. The current directory `DEFAULT_DATADIR` is pointing at which contains the `bitcoind.exe` file. 
+That file is what is necessary to start up a bitcoind node. 
+
+## Running Linux on a Windows Machine
+If you are looking to develop in a Linux environment on a Windows Machine I have had success with `Windows Subsystem for Linux (WSL)`. https://docs.microsoft.com/en-us/windows/wsl/install-win10
```
Comment (torkelrogstad) [inline docs/windows-users.md]:
```suggestion
If you are looking to develop in a Linux environment on a Windows Machine I have had success with [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
```

File: docs/windows-users.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Currently there are written changes in code to make it so you are run a node on Windows. Path specs are based on out of box installation. In the case you receive an error like `Could not locate bitcoind on user PATH`
+then you will need to do some tweaking of either the code or moving your folder into the correct location. Currently this is specified by the `DEFAULT_DATADIR` in `BitcoindConfig` within `Bitcoind-rpc`. When you install bitcoin out of box there are 2 folders
+created, 1 which is contained in the location on the wiki and that contains data generated after syncing. The current directory `DEFAULT_DATADIR` is pointing at which contains the `bitcoind.exe` file. 
+That file is what is necessary to start up a bitcoind node. 
+
+## Running Linux on a Windows Machine
+If you are looking to develop in a Linux environment on a Windows Machine I have had success with `Windows Subsystem for Linux (WSL)`. https://docs.microsoft.com/en-us/windows/wsl/install-win10   
+If you are interested in working on both Linux and Windows for development reasons I would also recommend `Windows Terminal` as a way to hold many different terminals all in one window including your
```
Comment (torkelrogstad) [inline docs/windows-users.md]:
```suggestion
If you are interested in working on both Linux and Windows for development reasons I would also recommend [Windows Terminal](https://github.com/microsoft/terminal) as a way to hold many different terminals all in one window including your

**Review by torkelrogstad — COMMENTED** [review]
What's your thoughts on moving this to a Windows section in the contribution guide?

**Review by Christewart — APPROVED** [review]

**Comment by cwaldron97** [comment]
I am up for the idea of adding it to the contribution guide. That would make the most sense. Although it might be difficult to break up sectioning of the document.

**Pull request merged**